### PR TITLE
Scenario 2021 with: Detailed Cavern, MB4 Neutron Shields (YB+2, YB-2,…

### PR DESCRIPTION
**Pull request by slomeo**
overlaps
@ianna  @fabiocos 
Scenario 2021 with: Detailed Cavern, MB4Shield and Muon overlaps fixed

#### PR description:

Scenario 2021 with: Detailed Cavern, MB4 Neutron Shields (YB+2, YB-2, S4 S13) and Overlaps, inside the Muon System, fixed

#### PR validation:

Test ok with cmsShow (there are 188 overlaps in CALO, TRACKER, FORWARDSHIELD)

#### if this PR is a backport please specify the original PR:

Before submitting your pull requests, make sure you followed this checklist:
- verify that the PR is really intended for the chosen branch
- verify that changes follow [CMS Naming, Coding, And Style Rules](http://cms-sw.github.io/cms_coding_rules.html)
- verify that the PR passes the basic test procedure suggested in the [CMSSW PR instructions](https://cms-sw.github.io/PRWorkflow.html)

### CODE REVIEW

**Comment by kpedro88** [comment]
please test

**Comment by cvuosalo** [comment]
+1

File: Geometry/CMSCommonData/python/cmsExtendedGeometry2021XML_cfi.py [hunk 1]
```diff
@@ -7,20 +7,23 @@
         'Geometry/CMSCommonData/data/materials.xml',
         'Geometry/CMSCommonData/data/rotations.xml',
         'Geometry/CMSCommonData/data/extend/v2/cmsextent.xml',
-        'Geometry/CMSCommonData/data/cms/2019/v2/cms.xml',
+#        'Geometry/CMSCommonData/data/cms/2019/v2/cms.xml',
```
Comment (kpedro88) [inline Geometry/CMSCommonData/python/cmsExtendedGeometry2021XML_cfi.py]: please delete all commented-out lines in this file

Comment (slomeo) [reply Geometry/CMSCommonData/python/cmsExtendedGeometry2021XML_cfi.py]: > please delete all commented-out lines in this file

done

**Comment by fabiocos** [comment]
@slomeo for what I can see, in the new cms.xml apart for the change of ```MBarRmax```, ```<Constant name="TotemBeamR1"  value="[MuonBeamR1]"/>``` disappears. Is this desired?

**Comment by fabiocos** [comment]
please test

**Comment by slomeo** [comment]
@fabiocos I added a line to cms.xml. Tks!

**Comment by cvuosalo** [comment]
+1

**Comment by fabiocos** [comment]
@kpedro88 any further comment?

**Comment by kpedro88** [comment]
+upgrade

**Comment by fabiocos** [comment]
+1

**Pull request merged**